.travis.yml

```diff
@@ -3,15 +3,18 @@ dist: xenial
 group: stable
 
 language: scala
-jdk: openjdk8
 
-apt:
-  packages:
-    - openjdk8
-
-before_install:
-  - PATH=$(echo "$PATH" | sed -e 's/:\/usr\/local\/lib\/jvm\/openjdk11\/bin//')
-  - JAVA_HOME=/usr/lib/jvm/java-1.8.0-openjdk-amd64
+## add the following jobs to the build matrix:
+matrix:
+  include:
+  - jdk: openjdk8
+    before_install:
+      - PATH=$(echo "$PATH" | sed -e 's/:\/usr\/local\/lib\/jvm\/openjdk11\/bin//')
+      - JAVA_HOME=/usr/lib/jvm/java-1.8.0-openjdk-amd64
```

Comment:
Is this necessary?  I see it was already there, but I've not had to do anything like this in JDK matrices.

Reply:
There used to be an issue specific to xenial not picking the right openjdk. Not sure if this had been addressed recently.
See for example https://github.com/travis-ci/travis-ci/issues/10290